entry_types/scrolled/package/spec/contentElements/dataWrapperChart/useIframeHeight-spec.js

```diff
@@ -0,0 +1,19 @@
+import {useIframeHeight} from '../../../src/contentElements/dataWrapperChart/useIframeHeight'
```

Comment:
Consider using an absolute path here, as we also do in [1]. This would decouple the spec file's content from its path

[1] https://github.com/codevise/pageflow/blob/master/entry_types/scrolled/package/spec/frontend/usePostMessageListener-spec.js#L1